README.md

```diff
@@ -1,16 +1,39 @@
-Portfolio setup and mobile skeleton
+## Portfolio Project
 
+On this project, I built personal portfolio
+
+To-do :
+ Mobile Skeleton
+ Mobile Cards
+ Desktop versions
+ Animations and transitions
+ Side menu for mobile
+ Modal Cards
+ Dynamic load
+ Form validation
+ Local storage
 
 ## Built With
 
 - HTML
 - CSS
+- Javascript
+
+# Prerequisites
+The basic requirements for building this project are:
+
+VSCode or any other equivalent code editor
+Node package manager
+
+# Live Demo:
+
+ekenecf.github.io
```

Comment:
- [ ] Right now this is text, let's change it to link.

Reply:
Hi @Deepakdanger. As requested by you,
1. I have made the changes on the readme file as I have changed the text to a link,
2. I have re-arranged the modal images.